test/library/packages/LinearAlgebra/performance/dot-perf.perfexecopts

```diff
@@ -1,7 +1 @@
---n=1000000 --iters=100 -nl 8 --reference=true #n1000000-8
---n=1000000000 --iters=10 -nl 8 --reference=true #n1000000000-8
---n=100000000000 --iters=2 -nl 8 --reference=true #100000000000-8
---n=1000000 --iters=20 -nl 4 --reference=true #n1000000-4
---n=1000000000 --iters=2 -nl 4 --reference=true #n1000000000-4
---n=1000000 --iters=20 -nl 2 --reference=true #n1000000-2
---n=1000000 --iters=10 -nl 1 --reference=true #n1000000-1
+--n=2048 --iters=10
```

Comment:
Any reason we dropped to 2048 from 4096?

Reply:
4096 was for distributed. This is for a single locale, so I thought 2048 would be a good size. Both the tests complete in about 30 seconds and 70 seconds respectively (if I remember correctly).

Reply:
Ah, that's fine then